docs/rst/manual/data_types/index.rst

```diff
@@ -0,0 +1,625 @@
+The data types available in ERT
+===============================
+
+Very briefly described the purpose of ERT is to pass uncertain paramater values
+to a simulator [#]_, in a form which works as suitable input to the simulator and
+then subsequently load the results from the simulator. This means that data must
+be formatted in a form which the simulator can load, and also that ERT must be
+able to read files generated by the simulator.
+
+The data managed by ERT are organized in different *data types* described in
+this chapter. Configuring the data used in the conditioning project is a very
+important part of setting up a ERT configuration file - in practical terms this
+is how you configure which uncertainty parameters should be studied. The data
+types in ert can be categorized in two ways:
+
+1. How the data type behaves dynamically: is it a static parameter like porosity
+   or a relperm parameter - i.e. does it serve as *input* to the simulator, or
```

Comment:
Could you stress the property of `relperm` here, to make the difference to `porosity` (being static) even more clear?